docs/workflow/blocks/context_block.rst

```diff
@@ -1,28 +1,44 @@
-Context
-=======
+Context & State
+===============
 
-Use the context block to save data into the global context to make it available for all
+Use the context block to save data into the flow context or global state to make it available for all
```

Comment:
Not just subsequent tasks but existing ones too. Not sure how best to say this.
@the-greenman